cmd/calyptia/create_aggregator_k8s.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/spf13/cobra"
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	cloud "github.com/calyptia/api/types"
+)
+
+var reHostnameRFC1123 = regexp.MustCompile(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$`)
+
+func newCmdCreateAggregatorOnK8s(config *config, testClientSet kubernetes.Interface) *cobra.Command {
+	var aggregatorName string
+	var noHealthCheckPipeline bool
+	var environmentID string
+	var tags []string
+
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	configOverrides := &clientcmd.ConfigOverrides{}
+
+	cmd := &cobra.Command{
+		Use:     "kubernetes",
+		Aliases: []string{"kube", "k8s"},
+		Short:   "Setup a new core instance on Kubernetes",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			if aggregatorName != "" && !validHostnameRFC1123(aggregatorName) {
+				return fmt.Errorf("invalid aggregator name %q, it must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character", aggregatorName)
+			}
+
+			created, err := config.cloud.CreateAggregator(ctx, cloud.CreateAggregator{
+				Name:                   aggregatorName,
+				AddHealthCheckPipeline: !noHealthCheckPipeline,
+				Version:                "", // TODO
```

Comment:
This is filled in by the calyptia-core instance, doesn't have to provided by the CLI.

Reply:
Right. I'll just remove the line.

Reply:
Done